src/Cake.Core/IO/ProcessArgumentBuilder.cs

```diff
@@ -83,6 +84,33 @@ public string FilterUnsafe(string source)
         }
 
         /// <summary>
+        /// Copies all the arhuments of the current <see cref="ProcessArgumentBuilder"/> to target <see cref="ProcessArgumentBuilder"/>.
+        /// </summary>
+        /// <param name="target">The copy target.</param>
+        public void CopyTo(ProcessArgumentBuilder target)
```

Comment:
Well, the person who got access to the token could just as well use the `Render` method, or am I missing something?